src/script.rs

```diff
@@ -29,6 +29,7 @@ use crate::{
 };
 
 const BASH_PREAMBLE: &str = r#"
+#!/bin/bash
```

Comment:
looks good but I think we need to put it even one line higher (right after the #")

Reply:
moved it up :)